lib/lp_token_auth/core.rb

```diff
@@ -1,4 +1,5 @@
 require 'jwt'
+require 'jwe'
```

Comment:
Looks promising! Is there an additional `gem` dependency we need to include?

Reply:
We will need the [jwe gem](https://github.com/jwt/ruby-jwe). Its made by the same people that made the jwt gem.